source/external-services/nextcloud/install-nextcloud.rst

```diff
@@ -0,0 +1,209 @@
+
+Die Datei Dockerfile
+--------------------
+
+.. code-block:: console
+
+  FROM nextcloud:stable
+  RUN apt-get update && apt-get install -y smbclient libsmbclient-dev && pecl install smbclient && docker-php-ext-enable smbclient && rm -rf /var/lib/apt/lists/*
+  
+Wenn du experimentierfreudig bist, kannst du statt `stable` auch `latest` schreiben.
+
+Mit der zweiten Zeile werden die Vorbereitungen für die Einbindungen der Home-Verzeichnisse (Samba-Shares) durchgeführt.
+
+Die Datei db.env
+----------------
+
+.. code-block:: console
+
+  MYSQL_ROOT_PASSWORD=geheim
+  MYSQL_PASSWORD=geheim
+  MYSQL_DATABASE=nextcloud
+  MYSQL_USER=nextcloud
+  
+Hier sind die Zugangsdaten für die Datenbank hinterlegt.
+
+Die Datei docker-compose.yml
+----------------------------
+
+.. code-block:: console
+
+    version: '3'
+
+  services:
+    db2:
+      image: mariadb
+      command: --transaction-isolation=READ-COMMITTED --binlog-format=ROW
+      restart: always
+      volumes:
+        - ./db:/var/lib/mysql
+      env_file:
+        - db.env
+  
+    redis2:
+      image: redis:alpine
+      restart: always
+  
+    app2:
+      build:
+        context: .
+        dockerfile: Dockerfile
+      restart: always
+      ports:
+        - 7771:80
+      volumes:
+        - ./nextcloud:/var/www/html
+      environment:
+        - MYSQL_HOST=db2
+        - REDIS_HOST=redis2
+      env_file:
+        - db.env
+      depends_on:
+        - db2
+        - redis2
+  
+    cron2:
+      build:
+        context: .
+        dockerfile: Dockerfile
+      restart: always
+      volumes:
+        - ./nextcloud:/var/www/html 
+      entrypoint: /cron.sh
+      depends_on:
+        - db2
+        - redis2
+  
+  volumes:
+    db:
+    nextcloud:
+    
+In der Datei `docker-compose.yml` werden die Services deiner Nextcloud beschrieben. 
+
+Das Verzeichnis `/var/www/html` des Webservers wird unter dem Verzeichnis `/srv/docker/nextcloud/nextcloud` auf dem Dockerhost abgelegt. Und das Datenverzeichnis `/var/lib/mysql` der Maria Datenbank wird unter dem Verzeichnis `/srv/docker/nextcloud/db` auf dem Dockerhost abgelegt.
+
+Damit sind alle Daten im Verzeichnis `/srv/docker/nextcloud`.
+
+Wenn du im Verzeichnis `/srv/docker/nextcloud` bist, startest du die Nextcloud mit ``docker-compose up -d --build``. 
+
+Jetzt must du mit einem Browser die Startseite `https://nextcloud.meine-schule.de` deiner neuen Nextcloud aufrufen und einen Benutzernamen und ein Passwort für den Nextcloud-admin angeben.
+
+.. image:: media/install-01.png
+   :alt: Server - Einstellungen
+   :align: center
+
+Da die Nextcloud hinter dem nginx-Proxy liegt und nicht weiß, ob die Benutzer die Nextcloud über http oder https aufrufen, wird eine Anmeldung über eine Nextcloud-Client-App scheitern. Mit einem Eintrag in `/srv/docker/nextcloud/nextcloud/config/config.php` kannst du das Problem lösen:
+
+.. code-block:: console
```

Comment:
The same as before

Reply:
Ok, I replaced the .. code-block:: console with .. code::

Reply:
Thanks, well done!